tests/vmcheck/test-layering-relayer.sh

```diff
@@ -93,6 +93,10 @@ echo "ok rebase"
 vm_assert_layered_pkg foo present
 echo "ok pkg foo relayered on rebase"
 
+vm_rpmostree rebase --skip-purge -b vmcheck_tmp/rebase_test
+vm_rpmostree cleanup -p
+echo "ok rebase branch syntax"
```

Comment:
Since it's not actually related to pkg layering, maybe this belongs more in `test-basic.sh` ?